src/test/scala/mimir/algebra/IntervalSpec.scala

```diff
@@ -0,0 +1,54 @@
+package mimir.algebra
+
+import org.specs2.mutable._
+import mimir.parser._
+import mimir.algebra._
+import mimir.algebra.function.FunctionRegistry
+import mimir.optimizer._
+import mimir.optimizer.expression._
+import mimir.test._
+
+object IntervalSpec extends Specification with RASimplify {
+
+
+  def expr = ExpressionParser.expr _
+
+
+  "The Interval" should {
+      "INTERVAL" >> {
+	simplify("INTERVAL('P1Y2M3W4DT5H6M7.008S')") must be equalTo(IntervalPrimitive(1,2,3,4,5,6,7,8))
+      }
+
+
+      "INTERVALPLUSINTERVAL" >> {
+        simplify("INTERVALPLUSINTERVAL(INTERVAL('P1Y2M3W4DT5H6M7.008S'), INTERVAL('P0Y1M1W1DT0H0M0S'))") must be equalTo(IntervalPrimitive(1,3,4,5,5,6,7,8))
```

Comment:
Shouldn't this be `INTERVAL('P1Y2M3W4DT5H6M7.008S') + INTERVAL('P0Y1M1W1DT0H0M0S')`?

Same for the following lines.